modified_files/net/net.gypi

```diff
@@ -817,6 +817,8 @@
       'http2/hpack/decoder/hpack_block_decoder.h',
       'http2/hpack/decoder/hpack_decoder_string_buffer.cc',
       'http2/hpack/decoder/hpack_decoder_string_buffer.h',
+#      'http2/hpack/decoder/hpack_decoder_tables.cc',
```

Comment:
reverted removal of hpack_decoder_tables.* files.
From https://codereview.chromium.org/2606733004, it seems that these new tables are not in use yet even in http2.